tools/packaging/static-build/virtiofsd/build-static-virtiofsd.sh

```diff
@@ -40,4 +40,43 @@ pull_virtiofsd_released_binary() {
     popd
 }
 
-pull_virtiofsd_released_binary
+init_env() {
+   case ${ARCH} in
+     "aarch64")
+       LIBC="musl"
+     ;;
+     "ppc64le")
+       LIBC="gnu"
+     ;;
+     "s390")
+       LIBC="gnu"
+  esac
+
+    ARCH_LIBC=${ARCH}-linux-${LIBC}
+}
+	   
+build_virtiofsd_from_source() {
+   echo "build viriofsd from source"
+   init_env
+
+   virtiofsd_url=$(get_from_kata_deps "externals.virtiofsd.url")
+
+   git clone ${virtiofsd_url} virtiofsd
```

Comment:
```suggestion
   git clone --depth 1 --branch ${virtiofsd_version} ${virtiofsd_url} virtiofsd
```

This way we clone the version we want to rely on.